src/Xhgui/Saver.php

```diff
@@ -41,7 +41,7 @@ public static function factory($config)
 
             case 'mongodb':
             default:
-                $mongo = new MongoClient($config['db.host'], $config['db.options']);
+                $mongo = new MongoClient($config['db.host'], $config['db.options'], $config['db.driverOptions']);
```

Comment:
This might also need to be added to the [xhgui-collector](https://github.com/perftools/xhgui-collector/blob/master/config/config.default.php) package as well to maintain parity.

Reply:
this is collector project :)